### PR TITLE
Update openapi3 file to add the content

### PR DESCRIPTION
**Pull request by YassinHajaj**
Adding the request body's content with application/offset+octet-stream

### CODE REVIEW

File: OpenAPI/openapi3.yaml [hunk 1]
```diff
@@ -81,6 +81,11 @@ paths:
       requestBody:
         description: (Possibly partial) content of the file. Required if Content-Length > 0.
         required: false
+        content:
+          application/offset+octet-stream":
```
Comment (Acconut) [inline OpenAPI/openapi3.yaml:85]: The double quote at the end of this line looks suspicious. Is this intended?

File: OpenAPI/openapi3.yaml [hunk 2]
```diff
@@ -334,6 +339,11 @@ paths:
       requestBody:
         description: Remaining (possibly partial) content of the file. Required if Content-Length > 0.
         required: false
+        content:
+          application/offset+octet-stream":
+            schema:
```
Comment (Acconut) [inline OpenAPI/openapi3.yaml:344]: Here as well.

File: OpenAPI/openapi3.yaml (hunk 1 above)
Comment (YassinHajaj) [reply OpenAPI/openapi3.yaml:85]: Hello, thx for replying. No it's not intended indeed. Go figure how it got there lol.
I'll push a fix.

**Review by Acconut — APPROVED** [review]
Thank you very much!

**Pull request merged**